addon/templates/components/fixtable-grid.hbs

```diff
@@ -76,9 +76,13 @@
               {{#each columns as |column|}}
                 <td class={{column.cellClass}}>
                   {{#if column.component}}
-                    {{!-- render the passed-in component --}}
-                    {{component column.component dataRow=row columnDef=column}}
-                  {{else}}
+                        {{#if column.clickable}}
```

Comment:
This if-block works, but I don't love the duplication in the {{component}} markup where the only that differs is the click action. It's potentially error-prone since we have keep the markup in-sync in two places.

Instead, could we move the `{{#if column.clickable}}` logic into the `clickCell` action? If we don't want to propagate the action, we can decide that in the JavaScript.